pom.xml

```diff
@@ -159,19 +158,18 @@ limitations under the License.
         <dependency>
             <groupId>org.apache.commons</groupId>
             <artifactId>commons-lang3</artifactId>
-            <version>3.9</version>
+            <version>3.12.0</version>
         </dependency>
 
         <dependency>
             <groupId>com.google.code.gson</groupId>
             <artifactId>gson</artifactId>
-            <version>2.8.2</version>
+            <version>2.10.1</version>
```

Comment:
This can be replaced with the [gson api plugin](https://plugins.jenkins.io/gson-api/dependencies)